data/json/recipes/basecamps/recipe_groups.json

```
@@ -32,7 +32,16 @@
     "id": "all_faction_base_expansions",
     "building_type": "NONE",
     "recipes": [
-      { "id": "faction_base_farm_0", "description": "Farm", "om_terrains": [ "field" ] },
+      {
+        "id": "faction_base_farm_0",
+        "description": "Farm version 1. Shed, plots, some farming recipes",
```

```suggestion
        "description": "Farm version 1.  Shed, plots, some farming recipes",
```
Two spaces after full stop.